build.sbt

```diff
@@ -77,7 +77,7 @@ buildInfoKeys := Seq[BuildInfoKey](version)
 buildInfoPackage := "com.gu.contentapi.buildinfo"
 buildInfoObject := "CapiBuildInfo"
 
-val CapiModelsVersion = "9.2"
+val CapiModelsVersion = "9.7"
```

Comment:
I've released a newer version capi-models (to include vine/content atoms), can you bump to 9.8? Or I can make a new PR.